core/src/main/java/org/elasticsearch/index/query/BoolQueryBuilder.java

```diff
@@ -169,18 +289,14 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
         doXArrayContent("filter", filterClauses, builder, params);
         doXArrayContent("must_not", mustNotClauses, builder, params);
         doXArrayContent("should", shouldClauses, builder, params);
-        if (boost != -1) {
+        if (boost != 1.0f) {
             builder.field("boost", boost);
         }
-        if (disableCoord != null) {
-            builder.field("disable_coord", disableCoord);
-        }
+        builder.field("disable_coord", disableCoord);
         if (minimumShouldMatch != null) {
             builder.field("minimum_should_match", minimumShouldMatch);
         }
-        if (adjustPureNegative != null) {
-            builder.field("adjust_pure_negative", adjustPureNegative);
-        }
+        builder.field("adjust_pure_negative", adjustPureNegative);
```

Comment:
I am starting to see that the default boost doesn't get printed out but other default fields do. Makes sense to me but maybe we want to be consistent? I think we should have this a separate discussion, make some decision and do the same everywhere (I have the feeling we are not yet settled yet on one way or another)